napari_allencell_annotator/util/file_utils.py

```diff
@@ -18,7 +19,21 @@ def select_only_valid_files(file_list: list[Path]) -> list[Path]:
         file_list: list[Path]
             A list of paths
         """
-        return [file for file in file_list if not file.name.startswith(".") and file.is_file()]
+        valid_files: List[Path] = []
+        for file in file_list:
+            if not file.name.startswith("."):
+                # get files
+                if file.is_file():
+                    valid_files.append(file)
+                # get zarr file
+                elif file.is_dir() and file.name.endswith(".zarr"):
+                    valid_files.append(file)
+                # get zarr inside the directory
+                else:
+                    dir_zarr_files: List[Path] = list(file.glob("*.zarr"))
+                    valid_files += dir_zarr_files
+
```

Comment:
this logic needs to move out of this function

Reply:
refactored in https://github.com/aics-int/napari-allencell-annotator/pull/79/commits/9e11e76dba1ee2d12f1f0fccd3b34dcfa4045f13